projectatomic/osbs-client
# render_add_filesystem: improved multi-arch build support

- Pass along an architectures variable to atomic-reactor
- Add new filesystem_koji_task_id parameter to create_worker_build API
- Bump DEFAULT_ARRANGEMENT_VERSION to 2
- Update self-tests with filesystem_koji_task_id usage
- Set from_task_id in render_add_filesystem, make use of set_label
- Make sure filesystem_koji_task_id actually propagates to worker builds

Successfully tested with matching atomic-reactor changes in staging
environment, with both the new arrangement version 2 and the prior
(slightly modified) version 1.
